tools/run_tests/run_xds_tests.py

```diff
@@ -569,6 +569,8 @@ def test_traffic_splitting(gcp, original_backend_service, instance_group,
     try:
         # Patch urlmap, change route action to traffic splitting between
         # original and alternate.
+        logger.info('disabling validateForProxy in target proxy')
+        set_validate_for_proxyless(gcp, False)
```

Comment:
If possible, can you also add a note about why this is necessary for the test to pass, and when/what conditions would enable removing this line in the future?

Reply:
Done